benchmark_runner/common/ocp_resources/create_ocp_resource_operations.py

```diff
@@ -12,6 +12,9 @@ class CreateOCPResourceOperations:
     """
     This class creates OCP resources
     """
+    # Number of CSVs that should be validated
+    ODF_CSV_NUM = 4
```

Comment:
You have the CSV names, correct?  Check them by name (make sure that all of the expected CSVs are validated) rather than count.

Reply:
I prefer to keep it generic as we can to avoid failures
We just need to install ODF and not to test it.

Reply:
@RobertKrawitz, I have fixed.